src/script/auth/page/ConversationJoin.tsx

```diff
@@ -151,11 +135,11 @@ const ConversationJoinComponent = ({
       await doRegisterWireless(
         registrationData as RegisterData,
         {
-          shouldInitializeClient: !isPwaEnabled,
+          shouldInitializeClient: true,
         },
         entropyData,
       );
-      const conversationEvent = await doJoinConversationByCode(conversationKey, conversationCode);
+      const conversationEvent = await doJoinConversationByCode(conversationKey ?? '', conversationCode ?? '');
```

Comment:
How about throwing an error early when those are not defined instead of putting empty string? Not very likely to happen, but doJoinConversationByCode will fail anyway when provided with empty strings. Reading catch method seems like simple error throw should make user get gracefully logged out.

Reply:
seems reasonable. im not clear if this is even a real risk but ill update it to throw an error.